prow/plugins/docs-no-retest/docs-no-retest_test.go

```diff
@@ -0,0 +1,303 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docsnoretest
+
+import (
+	"errors"
+	"testing"
+
+	"k8s.io/test-infra/prow/github"
+)
+
+type ghc struct {
+	*testing.T
+	labels    map[github.Label]bool
```

Comment:
Just store the label name instead of the entire `Label` struct since the `Color` and `ID` fields are ignored.